repos/system_upgrade/el7toel8/libraries/overlaygen.py

```diff
@@ -0,0 +1,142 @@
+import contextlib
+import os
+import shutil
+
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.common import mounting, utils
+from leapp.libraries.stdlib import CalledProcessError, api, run
+from leapp.models import XFSPresence
+
+
+def _logger():
```

Comment:
Because at import time, current_logger is not yet reachable - The actor that imports this library, has not been instantiated

Reply:
Ok well current_logger itself is reachable, but the instance of the logger it would return is not.
It would default to some fallback logger. I rather could import api.current_logger as _logger